src/blocks/server.blocks/middleware/__page-title/middleware__page-title.node.js

```diff
@@ -2,30 +2,40 @@ modules.define('middleware__page-title', ['config', 'logger', 'model'], function
 
     logger = logger(module);
 
-    provide(function () {
-        return function (req, res, next) {
-            logger.debug('get title by request %s', req.url);
+    provide({
+        run: function () {
```

Comment:
1. looks like this change should be supported somewhere else in the code.
2. are you sure that `run()` and `hasExceptions()` API is good?